src/renderer/ScreenController.tsx

```diff
@@ -341,17 +342,19 @@ export default class ScreenController extends Component {
           }}
         >
           <KeybindingsContextProvider>
-            <div className='main-container'>
-              <AccountListSidebar
-                selectedAccountId={this.selectedAccountId}
-                onAddAccount={this.addAndSelectAccount.bind(this)}
-                onSelectAccount={this.selectAccount.bind(this)}
-                openAccountDeletionScreen={this.openAccountDeletionScreen.bind(
-                  this
-                )}
-              />
-              {this.renderScreen()}
-            </div>
+            <ChatProvider>
```

Comment:
the sidebar does not need the context, does it?

Reply:
Yeah, that's right, it's not required. Dialogs need it though so it lives even lower now.